### ISSUE TITLE
Using typescript in .js

### ISSUE THREAD

**Issue by Tobjoern** [jondot/hygen#151]
I want to know more about the inner workings of hygen for learning purposes. That's why I cloned the repository, installed the dependencies and opened the project with vscode. To my dismay there are a lot of error messages in the source code, because typescript syntax is used in a .js file. 
I have also noticed this in other open source projects on github and I am wondering what the reason for this is (I assume so that it can be run without having to compile it to js first), and how you people develop this program further without being annoyed by constant error messages? Am I missing something?

![image](https://user-images.githubusercontent.com/31477665/65422976-301fb600-de08-11e9-9250-1034d3573a13.png)

(example of index.js)

**Comment by jondot** [comment]
Hi, Hygen actually uses [Flow](https://flow.org), not typescript; the errors are actually because something is causing your editor to think everything is Typescript. Flow is the gradual typing system that Facebook uses in production (and made by Facebook).

At some point in time Flow was a much better alternative to Typescript, and over a rather long period of time, it's now starting to be a consensus that Typescript is the better choice for projects looking forward; but I guess this is the kind of thing people can argue about forever.

In summary, not directly related to your experience, Hygen will be ported into Typescript in the immediate future.

For your learning purposes -- Flow and Typescript are virtually similar, so you can still learn how Hygen works. Of course, you have myself and the fantastic Hygen committers to help in every question, so feel free.

**Comment by Tobjoern** [comment]
Thanks for the quick reply! I have now installed flow and all the errors are gone! But if I install the flow extension (https://marketplace.visualstudio.com/items?itemName=gcazaciuc.vscode-flow-ide) a bunch of errors return because for instance the required return types aren't met properly. Is this intentional? Can I help with the conversion to typescript?

**Comment by jondot** [comment]
Closing since we moved to Typescript! 😄
